applications/BlockAIDS.md

```diff
@@ -0,0 +1,297 @@
+| 3. | Fungible token token balance display functionality | Users logged in with their wallet will be able to see Fungible token balance. User interface development: React.JS. Backend: NestJS. Development Language: Rust/Ink! |
+| 4. | Tansfer module | Trigger a payment by initiating a transfer, that will transfer Fungible token to other user's wallet. User interface development: React.JS. Backend: NestJS. Development Language: Rust/Ink!  | 
+| 5. | Withdraw module | Withdraw the Fungible token from our web app to user's own wallet. User interface development: React.JS. Backend: NestJS. Development Language: Rust/Ink! | 
+
+
+### Milestone 2 Loyalty and Reward Electronic System (LRES)
+
+- **Estimated duration:** 1.5 month
+- **FTE:**  4
+- **Costs:** 15,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one. 
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License  MIT  |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide Dockerfile and docker image to run the website in one command line, and you can to test those modules |
+| **0e.** | Artcile | We will publish an Artcile on Medium describing how to use Loyalty and Reward Electronic System |
+| 1. | Incentives management - Create | Create an incentive by user from predefined list of incentives. When the user completes the incentive, transfer module will be tiggered in order to transfer tokens, assigned to this task. Ink! smart contracts for implementing the logic of the Loyalty and Reward Electronic System |
+| 2. | Incentives management - Edit | Edit an incentive by user, by adding necessary details: deadline, organization & Specialist to which the incentive is assigned, Status, etc. Ink! smart contracts for implementing the logic of the Loyalty and Reward Electronic System |
+| 3. | Incentives management - Delete | Delete an incentive by user from the list of exustent incentives. Ink! smart contracts for implementing the logic of the Loyalty and Reward Electronic System |
+| 4. | Incentives management - Dashboard | An admin panel that allow to manage Loyalty and Reward Electronic System by web app admins. User interface development: React.JS. Backend: NestJS | 
+| 5. | Incentives management - Calendar | A comfortable calendar that allows to view and manage incentives in a calendar view. User interface development: React.JS. Backend: NestJS | 
```

Comment:
M2 is currently very vague. This seems to be a pallet related to a bonus program, but I don't see enough technical details. Also, the game logic for all participants to act honest is lacking here.